components/ws-daemon/pkg/content/initializer.go

```diff
@@ -99,7 +99,7 @@ func collectRemoteContent(ctx context.Context, rs storage.DirectAccess, ps stora
 // RunInitializer runs a content initializer in a user, PID and mount namespace to isolate it from ws-daemon
 func RunInitializer(ctx context.Context, destination string, initializer *csapi.WorkspaceInitializer, remoteContent map[string]storage.DownloadInfo, opts RunInitializerOpts) (err error) {
 	//nolint:ineffassign
-	span, ctx := opentracing.StartSpanFromContext(ctx, "RunInitializer")
+	span, _ := opentracing.StartSpanFromContext(ctx, "RunInitializer")
```

Comment:
Past experience shows that it's better to keep `ctx` around. If it's used in the future tracing doesn't breaking because future us didn't think of adding `ctx` back in. This also explains the `nolint` comment.

Reply:
fixed